rkovalenko/RemoveElement.cs

```diff
@@ -0,0 +1,23 @@
+public class Solution {
+    public int RemoveElement(int[] nums, int val) {
+        var i = 0;
+        var j = nums.Length - 1;
+        while (i <= j)
```

Comment:
Very elegant solution, I was thinking of exit condition myself but said "fuck it" and ended up traversing till the end.